src/ctap/storage/key.rs

```diff
@@ -114,21 +140,19 @@ make_partition! {
     /// The number of PIN retries.
     ///
     /// If the entry is absent, the number of PIN retries is `Customization::max_pin_retries()`.
-    PIN_RETRIES = 2044;
+    FIRST_PIN_RETRIES = 2044;
 
     /// The PIN hash and length.
     ///
     /// If the entry is absent, there is no PIN set. The first byte represents
     /// the length, the following are an array with the hash.
-    PIN_PROPERTIES = 2045;
+    FIRST_PIN_PROPERTIES = 2045;
```

Comment:
Double checking: PIN properties can not be an array in one key, because that could be too long? 8 hashes of 16 byte + a PIN length could fit into one key on Nordic, at least.

Reply:
Because it might be harder to migrate when we ever want to change the structure of PIN_PROPERTIES. I can wait until Julien's PR to land though, which provides an elegant solution.

Reply:
We can submit as is, and then do a new PR that merges #531 and uses it.

Reply:
I think I can wait, that CL is quite ready too. I'll prefer not to reserve key fields in a commit and remove them in upcoming commit.

Reply:
done